### PR TITLE
Closes #92 Add My Preferences section to My Account screen

### LINKED ISSUE #92: Add My Preferences section to My Account screen
The goal is to add My Preferences section to the My account screen, and currently the only available option will show: _My Store: Name of the Selected Store_. When tapping on this cell, we take the user to the store selection screen.

The UI of the cell should match the one we used for orders and reservations.

From the design perspective, we'd need an icon to show next to the section title.

### PR DESCRIPTION
**Pull request by nikola-mladenovic**
Tapping on _My Store_ button will present store selection screen:

simulator screen shot nov 24 2016 11 22 59 am

### CODE REVIEW

File: Sunrise/ViewControllers/OrdersViewController.swift [hunk 1]
```diff
@@ -11,7 +11,10 @@ import Result
 class OrdersViewController: UIViewController {
```
Comment (cneijenhuis) [inline Sunrise/ViewControllers/OrdersViewController.swift]: Maybe `OrdersViewController` is not fitting so well anymore?

**Review by cneijenhuis — COMMENTED** [review]
The ticket says:
> When tapping on this cell, we take the user to the store selection screen.

Is this also part of this PR? Maybe I'm blind, but I don't see the code for selecting the store...

File: Sunrise/ViewControllers/OrdersViewController.swift (hunk 1 above)
Comment (nikola-mladenovic) [inline Sunrise/ViewControllers/OrdersViewController.swift]: Not a part of this PR, it'll be implemented for https://github.com/commercetools/commercetools-sunrise-ios/issues/89

**Comment by nikola-mladenovic** [comment]
Not a part of this PR, it'll be implemented for https://github.com/commercetools/commercetools-sunrise-ios/issues/89

File: Sunrise/ViewControllers/OrdersViewController.swift (hunk 1 above)
Comment (nikola-mladenovic) [reply Sunrise/ViewControllers/OrdersViewController.swift]: Good point, how about `AccountViewController `(and of course, the view model shall be renamed as well)?

Comment (cneijenhuis) [reply Sunrise/ViewControllers/OrdersViewController.swift]: Yep 👍

**Review by cneijenhuis — APPROVED** [review]
After the rename, LGTM then 👍

**Pull request merged**